updatecli/policies/open-release-pr/updatecli.d/open-release-pr.yaml

```diff
@@ -0,0 +1,164 @@
+name: Open a PR that prepares the policy for release
+pipelineid: "{{ .pipelineid }}"
+
+sources:
+  newestGitHubDraftReleaseVersion:
+    name: Extract version from newest GitHub draft release from release drafter
+    kind: githubrelease
+    spec:
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repo }}"
+      token: "{{ requiredEnv .github.token }}"
+      key: "title"
+      typefilter:
+        draft: true
+      versionfilter:
+        kind: latest
+    transformers:
+      - trimprefix: "v"
+
+  newestReleaseTagVersion:
+    name: Extract newest tag from newest GitHub release
+    # used for displaying a gh compare URL
+    # disablesourceinput: true # we continue on fail (we may not have a tag)
+    kind: githubrelease
+    spec:
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repo }}"
+      token: "{{ requiredEnv .github.token }}"
+      key: "title"
+      versionfilter:
+        kind: latest
+
+  cargoPackageName:
+    name: Extract cargo package name from Cargo.toml
+    kind: toml
+    scmid: "policy-repo"
+    dependson:
+      - condition#checkCargoTomlExists
+    spec:
+      file: "Cargo.toml"
+      key: "package.name"
+
+conditions:
+  checkCargoTomlExists:
+    name: Check if Cargo.toml exists
+    scmid: "policy-repo"
+    kind: file
+    disablesourceinput: true
+    spec:
+      file: Cargo.toml
+  checkCargoLockExists:
+    name: Check if Cargo.lock exists
+    scmid: "policy-repo"
+    kind: file
+    disablesourceinput: true
+    spec:
+      file: Cargo.lock
+
+targets:
+  updateMetadataAnnotationVersion:
+    name: update metadata.yml with new version
+    kind: yaml
+    scmid: "policy-repo"
+    sourceid: newestGitHubDraftReleaseVersion
+    spec:
+      searchpattern: true
+      file: "metadata.y*ml"
+      key: "$.annotations.'io.kubewarden.policy.version'"
+      # value: '{{ source  "newestGitHubDraftReleaseVersion" }}'
```

Comment:
don't we need this line too?

Reply:
No, when passing the `sourceid`  to this target it automatically uses it as the value. Dropping it.

Reply:
Yes by default if you specify a sourceid then the target depends on that sourceid